### PR TITLE
rotational_cipher.py

### PR DESCRIPTION
**Pull request by amymhaddad**

### CODE REVIEW

File: rotational_cipher.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+"""Provide the encryption result for a given text""" 
+
+import string
+LETTERS = string.ascii_lowercase
+LIST_PUNCTUATION = string.punctuation
+LIST_NUMBERS = string.digits
```
Comment (amymhaddad) [inline rotational_cipher.py]: @paulghaddad I tried to combine these lists using zip, but I had a hard time seeing if an element is in a zipped list (see line 17). Do I have to do something in particular? Originally for line 17, I wanted to see if letter was in my zipped list, but that didn't work. Thoughts?

File: rotational_cipher.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+def rotate(text, key):
+    """Tests to create an implementation of the rotational cipher"""
+
+    new_text = ""
+
+    list_letters = list(LETTERS)
+    list_letters.append(' ')
+
+    for letter in text:
+        if letter in LIST_PUNCTUATION or letter in LIST_NUMBERS:
```
Comment (paulghaddad) [inline rotational_cipher.py]: Nice!

File: rotational_cipher.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        else:
+            index = list_letters.index(letter.lower())
+            rotational_cipher_formula = (index + key) % 26
+            new_letter = list_letters[rotational_cipher_formula]
+            if letter.isupper():
+                new_text += new_letter.upper()
+            elif letter.islower():
+                new_text += new_letter.lower()
+            else:
+                if letter == ' ':
```
Comment (paulghaddad) [inline rotational_cipher.py]: I think you can toss this condition into your condition on line 17, which will just tack on the space.

Comment (amymhaddad) [reply rotational_cipher.py]: @paulghaddad I get an error when I move this conditional to line 17, but I'm not sure why?

**Pull request closed without merging**